### PR TITLE
added float type

### PR DESCRIPTION
**Pull request by AndreiDev**
added float type mapping to typeMap

### CODE REVIEW

**Comment by hesserp** [comment]
:+1:

**Comment by tammok** [comment]
:+1:

**Comment by mrteutone** [comment]
:+1:

File: src/main/scala/org/zalando/spark/jsonschema/SchemaConverter.scala [hunk 1]
```diff
@@ -33,6 +33,7 @@ object SchemaConverter {
   val typeMap = Map(
     "string" -> StringType,
     "number" -> DoubleType,
+    "float" -> FloatType,
```
Comment (ovolynets) [inline src/main/scala/org/zalando/spark/jsonschema/SchemaConverter.scala]: @AndreiDev thanks for the pull request, it is nice to know the library seems useful. One thing: could you add this to the test in SchemaConverterTest. Thanks

**Comment by AndreiDev** [comment]
@ovolynets added the test :)

File: src/main/scala/org/zalando/spark/jsonschema/SchemaConverter.scala (hunk 1 above)
Comment (AndreiDev) [inline src/main/scala/org/zalando/spark/jsonschema/SchemaConverter.scala]: @ovolynets done

Comment (ovolynets) [inline src/main/scala/org/zalando/spark/jsonschema/SchemaConverter.scala]: cool, thank you. @AndreiDev do you need the new release published any time soon?

**Comment by ovolynets** [comment]
👍

File: src/main/scala/org/zalando/spark/jsonschema/SchemaConverter.scala (hunk 1 above)
Comment (AndreiDev) [inline src/main/scala/org/zalando/spark/jsonschema/SchemaConverter.scala]: @ovolynets that would be great 👍

**Comment by hesserp** [comment]
:+1:

**Comment by ovolynets** [comment]
@tammok or @hesserp could you take care of bumping the version and publishing?

File: src/main/scala/org/zalando/spark/jsonschema/SchemaConverter.scala (hunk 1 above)
Comment (hesserp) [inline src/main/scala/org/zalando/spark/jsonschema/SchemaConverter.scala]: @AndreiDev version 0.4 including your pull request has been published.

Comment (AndreiDev) [inline src/main/scala/org/zalando/spark/jsonschema/SchemaConverter.scala]: @hesserp very much appreciated :)

**Pull request merged**